hoomd/md/pair/aniso.py

```diff
@@ -400,6 +400,10 @@ class ALJ(AnisotropicPair):
         Type: `hoomd.data.typeparam.TypeParameter` [`tuple` [``particle_types``,
         ``particle_types``], `dict`]
 
+    Note:
+        Parameters which end with i and j refer to the first and second particle
+        type in the parameter tuple, respectively.
```

Comment:
I don't understand this. HOOMD sets the pair parameters symmetrically. From the user perspective:
```
alj.params[('A', 'B')] = dict(epsilon=1.1, sigma_i=2, sigma_j=3, alpha=0)
assert alj.params[('B', 'A')] == dict(epsilon=1.1, sigma_i=2, sigma_j=3, alpha=0)
```
This documentation would indicate that `sigma_i` applies to 'A' and `sigma_j` applies to 'B' AND contradicts itself by stating that `sigma_i` applies to 'B' and `sigma_j` applies to 'A' in the 2nd line.

Reply:
Code that demonstrates the symmetric nature of the parameters:
https://github.com/glotzerlab/hoomd-blue/blob/5f9041a58a62457e5b68e87bce9a8f51896824b3/hoomd/md/AnisoPotentialPair.h#L347-L354

Reply:
This is true and my original addition to the documentation could have been better worded. The implementation of the ALJ potential is symmetric under the exchange of the labels `i` and `j`. In fact, its also symmetric under the exchange of the contact ratio labels. In other words
``` python
alj.params[('A', 'B')] = dict(epsilon=1.0, sigma_i=1.0, sigma_j=2.0, contact_ratio_i=0.2, contact_ratio_j=0.1, alpha=0)
```
Evaluates the same potential as 
``` python
alj.params[('A', 'B')] = dict(epsilon=1.0, sigma_i=1.0, sigma_j=2.0, contact_ratio_i=0.1, contact_ratio_j=0.2, alpha=0)
```
This is because internally the average of the two parameters is always used instead of the individual values. Internally it computes something like
```
sigma_ij = Scalar(0.5) * (_params.sigma_i + _params.sigma_j)
contact_sigma_ij = Scalar(0.5) * (_params.contact_sigma_i + _params.contact_sigma_j)
```
and then just works with those values instead of the original individual parameters.

When a user sets the parameters, however, they will naturally want to associate the `i` parameters with one particle type and the `j` parameters with another particle type. In fact, they need to do this in order to get accurate simulation results. As an example,
``` python
# one of these sigma/contact_sigma values physically represents the A type
alj.params[('A', 'B')] = dict(epsilon=1.0, sigma_i=1.0, sigma_j=2.0, contact_ratio_i=0.1, contact_ratio_j=0.2, alpha=0)

# the same sigma/contact_sigma values that represent the A type above must also show up in this dictionary
alj.params[('A', 'C')] = dict(...)
```

The best way to document the behavior is to make both points clear: the actual potential is symmetric w.r.t. labels `i` and `j`, but when setting parameters for all typepairs, the same `sigma`/`contact_sigma` values must be shared across different typepairs that share one particle type in order to get proper simulation results.

Reply:
Yes, it would be great to document exactly how the values in the given equation are computed from the user input parameters.